### DISCUSSION TITLE
Quarkus 3.0, our new major release, is here!

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#32914] [Quarkus Blog/Website]
# Quarkus 3.0, our new major release, is here!

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-0-final-released/

**Comment by liupu9** [comment]
Congratulations~

**Comment by GabrielFoxDev** [comment]
:heart:

**Comment by yangyuanqiang** [comment]
Well done~

**Comment by OndraZizka** [comment]
"Updating to Quarkus 3.0" - how come this does not employ Windup :)
But seriously... could have also been Windup rules.

**Reply by OndraZizka** [reply]
And also... good job, congrats - I'm gonna upgrade my unfinished tryout project :)

**Reply by maxandersen** [reply]
@OndraZizka the rules are done in collaboration with windup team. Windups cli is not easily embeddable as of today and it's usecase does not yet "fit" in a simple UX of dedicated updates for a framework. Being worked on though :)

**Comment by vsevel** [comment]
bravo
